Makefile

```diff
@@ -7,6 +7,11 @@ PEM := $(PEM_DIR)/cert.pem
 NAMESERVER := 8.8.8.8
 RESOLV_CONF_DIR := endpoint/endpointComp
 OUTPUT_BASE_DIR := output_base
+# Endpoint build target. The default intends to the platform of your development system.
+TARGET := development_host
+# Build test suite
+TESTS := false
+ENABLE_HTTPS := false
```

Comment:
I am afraid of the confusion about HTTPS. The connection between endpoint and TA server is taking HTTP by default. But, TA will connect to available IOTA nodes only with HTTPS as @HowJMay is working.
How about changing the option to `ENFORCE_EP_HTTPS` and appending high-level descriptions as well?
`EP` is the abbreviation of "endpoint."